java/ql/lib/ext/okhttp3.model.yml

```diff
@@ -6,6 +6,7 @@ extensions:
       - ["okhttp3", "OkHttpClient", True, "newCall", "(Request)", "", "Argument[0]", "request-forgery", "ai-manual"]
       - ["okhttp3", "OkHttpClient", True, "newWebSocket", "(Request,WebSocketListener)", "", "Argument[0]", "request-forgery", "ai-manual"]
       - ["okhttp3", "Request", True, "Request", "", "", "Argument[0]", "request-forgery", "manual"]
+      - ["okhttp3", "Request$Builder", False, "get", "()", "", "Argument[this]", "request-forgery", "ai-manual"] # this creates a GET request
```

Comment:
This is only setting the method of the request to GET, isn't it? It doesn't seem like it should be more dangerous than, say, a POST or a PUT.

```suggestion
```

Reply:
We may have misinterpreted what this does — we thought, when creating the model that it would EXECUTE the request. Thank you!